x-pack/plugins/ml/server/models/data_frame_analytics/index_patterns.ts

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { APICaller, SavedObjectsClientContract } from 'kibana/server';
+import { IndexPatternAttributes } from 'src/plugins/data/server';
+// import { findObjectByTitle } from 'src/plugins/saved_objects/';
+import { mlLog } from '../../client/log';
+export const SAVED_OBJECT_TYPES = {
```

Comment:
We also define these in `ml/server/models/data_recognizer/data_recognizer.ts`. Probably worth moving them out to the `common` folder  - in `ml/common/types/kibana.ts` perhaps? But is it being used here?